tests/test_model_integration.py

```diff
@@ -148,9 +155,20 @@ def test_load_use_serialized_tagger():
     loaded_model.predict([sentence, sentence_empty])
     loaded_model.predict([sentence_empty])
 
+    sentence.clear_embeddings()
+    sentence_empty.clear_embeddings()
 
-def test_train_load_use_classifier():
-    corpus = NLPTaskDataFetcher.fetch_data(NLPTask.IMDB)
+    loaded_model: SequenceTagger = SequenceTagger.load('pos')
+
+    loaded_model.predict(sentence)
+    loaded_model.predict([sentence, sentence_empty])
+    loaded_model.predict([sentence_empty])
+
+
+@pytest.mark.slow
+def test_train_load_use_classifier(results_base_path, tasks_base_path):
```

Comment:
Do the integration tests really need to be skipped on travis? They only make small models and are important for catching lots of errors.